src/zope/testrunner/runner.py

```diff
@@ -125,6 +127,13 @@ def __init__(self, defaults=None, args=None, found_suites=None,
         self.tests_by_layer_name = {}
 
     def ordered_layers(self):
+        if (self.options.processes > 1 and not self.options.resume_layer):
+            # if we want multiple processes, we need a fake layer as first
+            # to start spreading out layers/tests to subprocesses
+            # but only if this is not in the subprocess
+            yield (name_from_layer(FakeMultiProcessLayer),
+                   FakeMultiProcessLayer, FakeMultiProcessTests_suite())
+
```

Comment:
Tried first without ```self.options.resume_layer``` but that ended up in an infinite loop. Every subprocess ran the FakeMultiProcessLayer tests, then spawn a subprocess for the real tests and so on.